apps/frontend/src/app/PageCharacter/CharacterDisplay/Tabs/TabOptimize/index.tsx

```diff
@@ -457,6 +487,13 @@ export default function TabBuild() {
             </Trans>
           </MenuItem>)}
         </DropdownButton>
+        <BootstrapTooltip placement="top" title={t`notifyTooltip`}>
+          <span>
```

Comment:
Remove the span, it is only needed if the button could be disabled, but it can't be.